pkgs/applications/virtualization/qemu/utils.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation rec {
     cp "${qemu}/bin/qemu-img" "$out/bin/qemu-img"
```

Comment:
Are these all statically linked? Or how can these be copied without needing a dependency on qemu libs?

Reply:
they're not statically linked but they don't depend on qemu neither. The only reference to the original qemu derivation is just through (what I think is) debug symbols (although I haven't been able to actually figure out what two `/nix/store/...-qemu` strings were actually used for).

Reply:
Interesting. Then I think this is ok, hoping that someone will weigh in, will merge when you ping in a few days otherwise 🙂